refinery/tool_manager/tests.py

```diff
@@ -413,13 +413,26 @@ def test_invalid_workflow_step_annotation_c(self):
             )
 
     def test_generate_tool_definitions_management_command(self):
-        with mock.patch('tool_manager.utils.get_workflow_list',
-                        side_effect=mock_get_workflow_list):
+        with mock.patch(
+            'tool_manager.utils.get_workflow_list',
+            side_effect=[
```

Comment:
@hackdna: I could also  just nix the `mock_get_workflow_list()` and do the `json.loads()` in this `side_effects` array? thoughts?